client/db.go

```diff
@@ -423,9 +426,9 @@ func sendAndFill(send func(...proto.Call) error, b *Batch) error {
 // Upon completion, Batch.Results will contain the results for each
 // operation. The order of the results matches the order the operations were
 // added to the batch.
-func (db *DB) Run(b *Batch) error {
+func (db *DB) Run(b *Batch) *proto.Error {
```

Comment:
Hmm, ok. I'd feel more strongly about this if the `client` package was exposed to the user. But it's not anymore. So this is fine by me.